src/handlers/psp.ts

```diff
@@ -166,48 +176,51 @@ const initializeState = (config: Config): State => {
 
 const executeApiCalls = async (state: State): Promise<State> => {
   const { config, baseLogOptions, groupedSubscriptions } = state;
-
   const apiValuePromises = groupedSubscriptions.map(({ subscriptions, template, endpoint }) => {
     const apiCallParameters = abi.decode(template.templateParameters);
-    return retryGo(() =>
-      callApi({
-        oises: config.ois,
-        apiCredentials: config.apiCredentials,
-        apiCallParameters,
-        oisTitle: endpoint.oisTitle,
-        endpointName: endpoint.endpointName,
-      }).then(
-        ([logs, data]) =>
-          [logs, { templateId: template.id, apiValue: data, subscriptions }] as node.LogsData<{
-            templateId: string;
-            apiValue: ethers.BigNumber | null;
-            subscriptions: Id<Subscription>[];
-          }>
-      )
+
+    return go(
+      () =>
+        callApi(config, {
+          id: template.id,
+          airnodeAddress: subscriptions.find((s) => s.templateId === Object.keys(template)[0])!.airnodeAddress,
```

Comment:
@acenolaza  Would this work for the PSP airnodeAddress? I think otherwise we would need to change the types in Airnode.